frontend/src/scenes/dashboard/TileFiltersOverride.tsx

```diff
@@ -0,0 +1,127 @@
+                        dateTo={overrides.date_to ?? null}
+                        onChange={(from, to) => setDates(from, to)}
+                        makeLabel={(key) => (
+                            <>
+                                <IconCalendar />
+                                <span className="hide-when-small"> {key}</span>
+                            </>
+                        )}
+                    />
+                </div>
+
+                <div>
+                    <label className="text-sm font-medium mb-2 block">Properties</label>
+                    <PropertyFilters
+                        onChange={(properties) => setProperties(properties)}
+                        pageKey={`tile_${tile.id}_properties`}
+                        propertyFilters={overrides.properties ?? []}
+                        taxonomicGroupTypes={[
+                            TaxonomicFilterGroupType.EventProperties,
+                            TaxonomicFilterGroupType.PersonProperties,
+                            TaxonomicFilterGroupType.EventFeatureFlags,
+                            TaxonomicFilterGroupType.EventMetadata,
+                            ...groupsTaxonomicTypes,
+                            TaxonomicFilterGroupType.Cohorts,
+                            TaxonomicFilterGroupType.Elements,
+                            TaxonomicFilterGroupType.SessionProperties,
+                            TaxonomicFilterGroupType.HogQLExpression,
+                            TaxonomicFilterGroupType.DataWarehousePersonProperties,
+                        ]}
+                    />
+                </div>
+
+                <div>
+                    <label className="text-sm font-medium mb-2 block">Breakdown</label>
+                    <BindLogic logic={insightLogic} props={insightProps}>
+                        <TaxonomicBreakdownFilter
+                            insightProps={insightProps}
+                            breakdownFilter={overrides.breakdown_filter ?? null}
+                            isTrends={false}
+                            showLabel={false}
+                            updateBreakdownFilter={(breakdown_filter: BreakdownFilter) => {
+                                if (
+                                    !breakdown_filter.breakdown &&
+                                    !breakdown_filter.breakdowns &&
+                                    !breakdown_filter.breakdown_type
+                                ) {
+                                    setBreakdown(null)
+                                } else {
+                                    setBreakdown(breakdown_filter)
+                                }
+                            }}
+                            updateDisplay={() => {}}
+                            disablePropertyInfo
+                            size="small"
+                        />
+                    </BindLogic>
+                </div>
+            </div>
+
+            {hasOverrides && (
```

Comment:
This clear button looks out of place. Maybe put in the footer on the left? 
<img width="554" height="246" alt="Image" src="https://github.com/user-attachments/assets/118e9284-31ee-4576-9e4b-91809a663ce7" />